### PR TITLE
Keypad

### LINKED ISSUE #130: Keypad Controller
As a user, I shall be able to move my vehicle by pressing keys on a keypad in the app.

Acceptance criteria:

- The app has a keypad display with buttons
- Buttons include forward, backward, right, left and stop.

### PR DESCRIPTION
**Pull request by FaizaAmjad**
⛓ **Description:**

- Close #130 

Switch the joystick to the keypad controller with buttons.

🛠 **Type of change:**
- [x] Keypad display to control the vehicle.
- [x] Button includes forward, backward, right, left and stop.
- [x] Connect sketch and the app via MQTT
- [x] Fix obstacle avoidance

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

File: GreenGarbageApp/app/src/main/java/com/example/greengarbageapp/fragments/GameFragment.kt [hunk 1]
```diff
@@ -12,94 +11,112 @@ import androidx.navigation.fragment.findNavController
...
+
         binding.endGame.setOnClickListener {
-            val points = binding.count.text.toString().toInt()
+            val points = 0
             val distance = binding.distance.text.toString().toInt()
             val action = GameFragmentDirections.actionGameFragmentToEndFragment(distance, points)
             findNavController().navigate(action)
         }
 
-        binding.obstacleAvoid.setOnCheckedChangeListener { _, isChecked ->
```
Comment (Adrianmeda) [inline GreenGarbageApp/app/src/main/java/com/example/greengarbageapp/fragments/GameFragment.kt:42]: This method is needed 👍

Comment (FaizaAmjad) [reply GreenGarbageApp/app/src/main/java/com/example/greengarbageapp/fragments/GameFragment.kt]: I have added this function now.

**Comment by Toerngren** [comment]
Fixed merge conflicts in fragment game.xml

**Review by umairngo — APPROVED** [review]
Everything is working, code tested and application running. 
Some small changes made in the mod by me but this branch is ready to merge.

**Pull request merged**